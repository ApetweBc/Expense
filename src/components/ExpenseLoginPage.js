import React from 'react';
import { connect } from 'react-redux';
import { startLogin } from '../actions/auth';



const ExpenseLoginPage = ({ startLogin }) =>  (

    <div>
       <button onClick={startLogin}> Login </button>
    </div>
);

const mapDispatchToProps = (dispatch) => ({
       startLogin:  () => dispatch(startLogin())
});
export default connect (undefined, mapDispatchToProps)(ExpenseLoginPage);
