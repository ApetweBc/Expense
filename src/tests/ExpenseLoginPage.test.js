import React from 'react';
import {shallow} from 'enzyme';
import ExpenseLoginPage from '../components/ExpenseLoginPage';


test('should correctly render LoginPAge', ()=>{
   
    const wrapper = shallow (<ExpenseLoginPage />);
    expect (wrapper).toMatchSnapshot();
});