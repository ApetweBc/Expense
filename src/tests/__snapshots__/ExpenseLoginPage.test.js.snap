// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render LoginPAge 1`] = `
<div>
  <button>
     Login 
  </button>
</div>
`;
